include/swift/Basic/InlineBitfield.h

```diff
@@ -2,7 +2,7 @@
 //
 // This source file is part of the Swift.org open source project
 //
-// Copyright (c) 2017 Apple Inc. and the Swift project authors
```

Comment:
Having a range of 2017 - 2017 doesn't seem like the right change.  This is repeated a couple times in the diff.

Reply:
Do you think those should be `... (c) 2017 Apple ...` instead?

Reply:
Yes.

Reply:
Thanks @tkremenek, I might get to this next weekend (17th/18th), does Apple have needs on having this done sooner? cc @shahmishal